test/run-qemu

```diff
@@ -29,14 +36,23 @@ fi
 KVERSION=${KVERSION-$(uname -r)}
 
 VMLINUZ="/lib/modules/${KVERSION}/vmlinuz"
+if ! [ -f "$VMLINUZ" ]; then
+    VMLINUZ="/lib/modules/${KVERSION}/vmlinux"
+fi
 
 if ! [ -f "$VMLINUZ" ]; then
     [[ -f /etc/machine-id ]] && read -r MACHINE_ID < /etc/machine-id
 
     if [[ $MACHINE_ID ]] && { [[ -d /boot/${MACHINE_ID} ]] || [[ -L /boot/${MACHINE_ID} ]]; }; then
         VMLINUZ="/boot/${MACHINE_ID}/$KVERSION/linux"
-    else
+    elif [ -f "/boot/vmlinuz-${KVERSION}" ]; then
         VMLINUZ="/boot/vmlinuz-${KVERSION}"
+    elif [ -f "/boot/vmlinux-${KVERSION}" ]; then
+        VMLINUZ="/boot/vmlinux-${KVERSION}"
```

Comment:
Yes, on SUSE `ppc*` arch kernel is named `vmlinux` as well. And, also on SUSE distros, we have the following kernel names for the other architectures:
- `s390*` => `image`
- `aarch64` => `Image`
- `armv*` => `zImage`

But, I don't know if all this specific naming follows a standard convention, so it can be upstreamed.

Reply:
@aafeijoo-suse Should I extend the patch to also search for those files or will you provide a follow-up merge request?

Reply:
I'd like to know the kernel names used in Ubuntu/Debian and RH/Fedora (@lnykryn could you shed some light here?) for the different architectures, so we could decide to extend this patch based on the response.

Reply:
We name everything vmlinuz. /boot/vmlinuz-5.14.0-358.el9.s390x, /boot/vmlinuz-5.14.0-358.el9.aarch64,....

Reply:
On Debian/Ubuntu the kernel is named `vmlinuz` (in `/etc` and has a version suffix) on most architectures, but sometimes `vmlinux` (e.g. ppc64el). Example:
```
ls /boot/vmlinuz* -l
lrwxrwxrwx 1 root root       24 Aug 12 20:16 /boot/vmlinuz -> vmlinuz-6.2.0-27-generic
-rw------- 1 root root 14482632 Jul 10 22:31 /boot/vmlinuz-6.2.0-26-generic
-rw------- 1 root root 13872968 Jul 12 23:23 /boot/vmlinuz-6.2.0-27-generic
lrwxrwxrwx 1 root root       24 Aug 12 20:16 /boot/vmlinuz.old -> vmlinuz-6.2.0-26-generic
```

Reply:
hmm.. we cannot handle all these downstream naming differences... maybe the right approach here is to add the ability to set `VMLINUZ` using an external variable?

Reply:
AFAIK vmlinux vs vmlinuz is an accepted standard. The first should be uncompressed second one is compressed. So at least adding vmlinux makes sense.

Reply:
Ok, good enough for me. Thanks!